drivers/spi/Kconfig.xec_qmspi_v2

```diff
@@ -0,0 +1,12 @@
+# Microchip XEC QMSPI V2 supporting MEC172x
+
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config SPI_XEC_QMSPI_V2
+	bool "Microchip XEC QMSPI V2 driver"
+	default n
```

Comment:
Above, please add

```
DT_COMPAT_MICROCHIP_XEC_QMSPI_V2 := microchip,xec-qmspi-v2
```

then, here:

```
	default $(dt_compat_enabled,$(DT_COMPAT_MICROCHPI_XEC_QMSPI_V2))
```

This is generally the right thing to do for kconfig values. Saying `default n` is always redundant and makes things harder to use for your users, because then they have to enable CONFIG_SPI **and** your IP block specific Kconfig option.

Reply:
We have two SPI drivers for MEC172x, the latest version 2 was added to work with the Zephyr NOR driver. 
So the application can enable only one of the two XEC SPI drivers, CONFIG_SPI_XEC_QMSPI_LDMA or CONFIG_SPI_XEC_QMSPI_V2.